<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsView</class>
 <widget class="QDialog" name="SettingsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>230</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>230</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>440</width>
     <height>1</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="OkButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Left only</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Resources/LeftOnly.png</pixmap>
   </property>
  </widget>
  <widget class="QComboBox" name="LeftOnlyCbox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Nothing</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CopyLeft</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CopyRight</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OverrideLeft</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OverrideRight</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DeleteLeft</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DeleteRight</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Left new</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Resources/LeftNew.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Right new</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Resources/RightNew.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Right only</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Resources/RightOnly.png</pixmap>
   </property>
  </widget>
  <widget class="QComboBox" name="LeftNewCbox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Nothing</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CopyLeft</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CopyRight</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OverrideLeft</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OverrideRight</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DeleteLeft</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DeleteRight</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="RightNewCbox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Nothing</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CopyLeft</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CopyRight</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OverrideLeft</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OverrideRight</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DeleteLeft</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DeleteRight</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="RightOnlyCbox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Nothing</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CopyLeft</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CopyRight</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OverrideLeft</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OverrideRight</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DeleteLeft</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DeleteRight</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="CategoryLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Category</string>
   </property>
  </widget>
  <widget class="QLabel" name="ActionLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Action</string>
   </property>
  </widget>
  <widget class="QLabel" name="LeftOnlyLabel">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>110</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LeftNewLabel">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>110</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="RightNewLabel">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>110</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="RightOnlyLabel">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>110</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>LeftOnlyCbox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SettingsView</receiver>
   <slot>LeftOnlyCboxIndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LeftNewCbox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SettingsView</receiver>
   <slot>LeftNewCboxIndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RightNewCbox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SettingsView</receiver>
   <slot>RightNewCboxIndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RightOnlyCbox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SettingsView</receiver>
   <slot>RightOnlyCboxIndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OkButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsView</receiver>
   <slot>OkClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>LeftOnlyCboxIndexChanged(int)</slot>
  <slot>LeftNewCboxIndexChanged(int)</slot>
  <slot>RightNewCboxIndexChanged(int)</slot>
  <slot>RightOnlyCboxIndexChanged(int)</slot>
  <slot>OkClicked()</slot>
 </slots>
</ui>
